### PR TITLE
Use View events

### PR DESCRIPTION
**Pull request by bmarty**
This is a sanity rework.
No effective change on the UI/UX

A few questions for @BillCarsonFr as TODOs in the code :)

### CODE REVIEW

File: vector/src/main/java/im/vector/riotx/features/crypto/verification/VerificationBottomSheetViewEvents.kt [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+package im.vector.riotx.features.crypto.verification
+
+import im.vector.riotx.core.platform.VectorViewEvents
+
+/**
+ * Transient events for the verification bottom sheet
+ */
+sealed class VerificationBottomSheetViewEvents : VectorViewEvents {
+    // data class Loading(val message: CharSequence? = null) : RoomDirectoryViewEvents()
```
Comment (ganfra) [inline vector/src/main/java/im/vector/riotx/features/crypto/verification/VerificationBottomSheetViewEvents.kt]: I don't think we should let these lines... at least let them inherits the right class

File: vector/src/main/java/im/vector/riotx/features/settings/devices/VectorSettingsDevicesFragment.kt [hunk 2]
```diff
@@ -64,35 +62,19 @@ class VectorSettingsDevicesFragment @Inject constructor(
...
-                                        transactionId = action.transactionId!!
-                                ).show(childFragmentManager, "REQPOP")
-                            }
-                        }
-                    }
+                is DevicesViewEvents.Loading            -> showLoading(it.message)
+                is DevicesViewEvents.Failure            -> showFailure(it.throwable)
+                is DevicesViewEvents.RequestPassword    -> maybeShowDeleteDeviceWithPasswordDialog()
+                is DevicesViewEvents.PromptRenameDevice -> displayDeviceRenameDialog(it.deviceInfo)
+                is DevicesViewEvents.ShowVerifyDevice   -> {
```
Comment (BillCarsonFr) [inline vector/src/main/java/im/vector/riotx/features/settings/devices/VectorSettingsDevicesFragment.kt]: For the view event, the device id is not needed, the transactionId is enough

File: vector/src/main/java/im/vector/riotx/features/crypto/verification/VerificationBottomSheetViewEvents.kt (hunk 1 above)
Comment (bmarty) [reply vector/src/main/java/im/vector/riotx/features/crypto/verification/VerificationBottomSheetViewEvents.kt]: ok, removed

File: vector/src/main/java/im/vector/riotx/features/settings/devices/VectorSettingsDevicesFragment.kt (hunk 2 above)
Comment (bmarty) [reply vector/src/main/java/im/vector/riotx/features/settings/devices/VectorSettingsDevicesFragment.kt]: ok, thanks

**Pull request merged**